server/sonar-web/src/main/js/api/webhooks.ts

```diff
@@ -60,3 +56,56 @@ export function updateWebhook(data: {
     .catch(() => Promise.resolve())
     .catch(throwGlobalError);
 }
+
+export function searchDeliveries(data: {
+  ceTaskId?: string;
+  componentKey?: string;
+  webhook?: string;
+  p?: number;
+  ps?: number;
+}): Promise<{
+  deliveries: WebhookDelivery[];
+  paging: Paging;
+}> {
+  return getJSON('/api/webhooks/deliveries', data)
+    .catch(() => ({
+      deliveries: [
+        {
+          at: '12.02.2018',
+          durationMs: 20,
+          httpStatus: 200,
+          id: '2',
+          success: true
+        },
+        {
+          at: '11.02.2018',
+          durationMs: 122,
+          httpStatus: 500,
+          id: '1',
+          success: false
+        }
+      ],
+      paging: {
+        pageIndex: 1,
+        pageSize: 10,
+        total: 15
+      }
+    }))
+    .catch(throwGlobalError);
+}
+export function getDelivery(data: {
```

Comment:
empty line ^

Reply:
we should have a formatting rule for that kind of stuff

Reply:
I'll check

Reply:
something like this maybe : https://eslint.org/docs/rules/lines-between-class-members

Reply:
here you go: https://github.com/SonarSource/sonarqube/pull/3043

Reply:
Cool! Thanks!